evaluator/evaluator.py

```diff
@@ -16,6 +16,7 @@
 from common.database_handler import DatabasePool, DatabasePoolConnection
 from common.logging import init_logging, get_logger
 from common.failed_cache import FailedCache
+from common.mqueue import PAYLOAD_TRACKER_TOPIC
```

Comment:
there is mqueue import above, no need to import part of the module again, just use mqueue.PAYLOAD_TRACKER_TOPIC